core/src/test/java/org/springframework/security/core/userdetails/PasswordEncodedUser.java

```diff
@@ -23,7 +23,7 @@
  * @since 5.0
  */
 public class PasswordEncodedUser {
-	private static final UserDetails USER = withUsername("user").password("password").roles("USER").build();
+	private static final UserDetails USER = withUsername("User").password("password").roles("USER").build();
```

Comment:
I understand what you are trying to say, however this does not change other test assumptions because this variable is only used as a variable returned from mocks.
IMHO It's not a good idea to increase the unit test batteries for trivial cases like this one (It doesn't event change the cyclomatic complexity) since they will have to be maintained without adding more value to the code. I thought that even if now I don't impact the tests globally, it's interesting to make the change here to keep in mind that usernames may not be in lower case always.
Perhaps I've made too many assumptions and didn't explain them. 
Anyway, I have no objection to following your suggestion. Thank you very much.